### ISSUE TITLE
Add support for latest versions of packages

### ISSUE THREAD

**Issue by sshishov** [edx/edx-lint#61]
Packages:
  - astroid-1.6.1
  - pylint-1.8.2
  - pylint-django-0.9.0

This is by today. Tomorrow it may change. But it is better to be up to date with latest pylint

**Comment by sshishov** [comment]
@nedbat agree, it is difficult to support the latest 3rd party project. But you can start supporting it from some version and if someone is using old version, let them install old package of `edx-lint`.

btw, when do you plan to enable this `dependabot`?

**Comment by nedbat** [comment]
These days, we aren't looking at dependabot, but we have other ideas for how to stay current.  The next thing to do is to make this repo conform to [OEP-18](https://open-edx-proposals.readthedocs.io/en/latest/oep-0018-bp-python-dependencies.html)
